src/adventuretime.coffee

```diff
@@ -14,14 +14,22 @@
 #   Christina[@ChrisBin]
 
 data = require './data.json'
+mitto = require 'mitto'
+config = mitto.loadConfig('adventuretime.config')
 
 module.exports = (robot) ->
+  count = robot.brain.get 'messageCount'
+  if !count then robot.brain.set 'messageCount', 0
   # Add response when hearing corresponding font
   addResponse = (d) ->
     regexTemplate = '.*\\b' + d.prompt.join('\\b|\\b') + '\\b.*'
     regex = new RegExp regexTemplate, 'i'
     robot.hear regex, (msg) ->
-      msg.send msg.random d.response
+      count = robot.brain.get 'messageCount'
+      count++;
```

Comment:
Since messageCount is just an integer, isn't this just updating the value in the variable `count` rather than what is in the brain?
I think you need to add this line after it:
```
robot.brain.set 'messageCount', count 
```
otherwise the messageCount will always be 0